NCS.DSS.AzureSearchUtility/Helpers/IndexerHelper.cs

```diff
@@ -41,7 +40,7 @@ private static void DeleteIndexer(SearchServiceClient searchService, Indexer ind
             var exists = searchService.Indexers.ExistsAsync(indexer.Name).GetAwaiter().GetResult();
```

Comment:
Might be worth reviewing this line with focus on usage of GetAwaiter and GetRestult. Something doesn't look quite right.